pkg/handlers/generic/mutation/imageregistries/credentials/inject_test.go

```diff
@@ -161,7 +161,7 @@ var _ = Describe("Generate Image registry patches", func() {
 					Path:      "/spec/template/spec/kubeadmConfigSpec/files",
 					ValueMatcher: gomega.ContainElements(
 						gomega.HaveKeyWithValue(
-							"path", "/etc/cre/install-kubelet-credential-providers.sh",
+							"path", "/etc/caren/install-kubelet-credential-providers.sh",
```

Comment:
should we reuse constants everywhere instead of specifying paths directly (even in tests)

Reply:
In https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/656/commits/1c618d607dd6ece68645845276c9a61fbc962813, I added common code to derive the path.

However, I think it is better to not use this code in tests. If we unintentionally change the path, we want the tests to fail, and not using the code ensures this. (On the other hand, if we intentionally change the path, we can update the tests using search/replace.)